core/server/master/src/main/java/alluxio/underfs/MasterUfsManager.java

```diff
@@ -98,8 +99,10 @@ public void addMount(long mountId, final AlluxioURI ufsUri,
                        final UnderFileSystemConfiguration ufsConf) {
     super.addMount(mountId, ufsUri, ufsConf);
 
-    try {
-      for (String physicalUfs : get(mountId).getUfs().getPhysicalStores()) {
+    try (CloseableResource<UnderFileSystem> ufsClientResource =
```

Comment:
A minor style comment - we could shorten the variable name to make the formatting nicer. Not sure if there is a good, short name.